src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -114,6 +114,7 @@
     <Compile Remove="$(PathToMsalSources)\Platforms\**\*.*;$(PathToMsalSources)\Resources\*.cs" />
     <Compile Remove="$(PathToMsalSources)\PlatformsCommon\PlatformNotSupported\ApiConfig\SystemWebViewOptions.cs" />
     <EmbeddedResource Include="$(PathToMsalSources)\Properties\Microsoft.Identity.Client.rd.xml" />
+    <None Include="$(PathToMsalSources)\..\..\..\README.md" Pack="true" PackagePath="\"/>
```

Comment:
will this be packed into the NuGet? all packages?

Reply:
No, this is included in `MSAL.csproj` so it only affects MSAL.